internal/clients/interface.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/RHEnVision/provisioning-backend/internal/models"
 )
 
+type AzureResourceGroup struct {
+	ID       string
+	Name     string
+	Location string
```

Comment:
You removed bunch of locations to region, shall we remove this one as well?

Reply:
I went back on the decision, lets keep Location, but explain it's region, not region with availability zone, as it souns like it the first look.